### PR TITLE
Correct date formatting

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Following [this work](https://github.com/DemocracyClub/yournextrepresentative/pull/1923) which introduced a `FORMAT_MODULE_PATH`, the date format has changed to MM/DD/YYYY when it should be DD/MM/YYYY.  This work turns changes the settings `USE_L10N = True` and corrects the path requirements of the `FORMAT_MODULE_PATH`.

Home page election list

Screenshot 2022-11-30 at 12 36 23 PM

Moderation page

Screenshot 2022-11-30 at 2 06 50 PM

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
[YNR timedate fix](https://trello.com/c/gEYTMytc/3146-ynr-timedate-fix)

File: ynr/settings/base.py [hunk 1]
```diff
@@ -412,7 +412,7 @@ def root(*path):
 
 RESULTS_FEATURE_ACTIVE = True
 CAN_EDIT_ELECTIONS = False
-FORMAT_MODULE_PATH = "settings.formats"
+FORMAT_MODULE_PATH = "ynr.formats"
```
Comment (symroe) [inline ynr/settings/base.py:415]: Is this right? That module doesn't exist, I don't think?

Comment (VirginiaDooley) [reply ynr/settings/base.py:415]: I followed this format https://docs.djangoproject.com/en/4.1/topics/i18n/formatting/#creating-custom-format-files and, it's being found this way.

Comment (symroe) [reply ynr/settings/base.py:415]: In that example, the `formats` file is directly inside `mysite` (in the directory tree code block). We have `formats in `ynr/settings/constants/`, so the python path would be `ynr.settings.constants.formats`

**Review by Bekabyx — APPROVED** [review]

**Review by symroe — APPROVED** [review]

**Pull request merged**